meeting_notes/20161112_planning-meeting-notes.md

```diff
@@ -0,0 +1,173 @@
+  - some interest in the tomesh.mini game, invitation to conference around games and outreach
+
+2. Rotman MBA students looking at tomesh (ethan and anastasia)
+  - [**Presentation Slides**](https://drive.google.com/file/d/0B46AXvjqbNM3UWI1U3JQdjRHWVk/view?usp=sharing)
+  - question posed as: "how do people percieve toronto mesh"
+  - preliminary impressions from presentations:
+  - really focused on the service (e.g., cost-savings) and options from the status-quo
+  - emphasis on onboarding and access through the website
+  - how the pi was perceived (or a piece of hardware) and emphasizing design (https://en.enigmabox.net/)
+  - more concise summary to follow, for now we whould think of:
+  - focused mandate to get people on board
+  - paying attention to onboarding
+  - segmentation of potential tomesh users
+
+3. Prototype (benhylau)
+  - ongoing work on getting the prototype up and running, #virtualmesh nights are when that happens
+  - increasing alignment with ansuz's projects
+  - down the road looking at metrics, including the friefunk, based on conversations with lars (e.g. prometheus, ...)
+
+4. Website (garry and dcwalk)
+  - there is a new website!
+  - met as working group to create new version
+
+### Previous Planning Session
+
+From [August 05](https://github.com/tomeshnet/documents/blob/master/meeting_notes/20160805_planning-meeting-notes.md):
+- **DONE** plan the series of workshops
+  - prototype hardware
+  - educational sessions?
+  - current status of Toronto infrastructure, and how it's structured.
+- _PARKED to 2017_ reach out to Wireless Slovenia
+- _PARKED to 2017_ plan international mesh event in Toronto
+- **DONE** revise Vision Statement by next weekend (voted to have Curtis draft it, followed by a week of review on the PR)
+- _BLOCKED_ form Code of Conduct Working Group (Lee, Nick, Ben, Leandro, Dennis, David, Curtis, Michael, dcwalk)
+- **DONE** deploy Vector.im and form virtual space to collaborate with other meshes (Garry, Ben)
+- **DONE** investigate Kune (pre-alpha collaboration software)
+- **DONE** figure out relationship with non-tech group over next few weeks
+  - Upcoming TCN workshop: Mesh Introduction on Fri, Nov 18
+- **DONE** start building a virtual mesh among all tomesh people, and eventually create a physical mesh taking consideration of people's privacy concerns over exposing home addresses
+- _PARKED_ figure out whether we need some legal status and how to acquire it
+
+## Current Decisions
+
+### Logo
+
+- We have [placeholder logo](https://tomesh.net/images/tomesh-icon.svg)) designed by garry already on website, chat, and github account
+- William designed a few versions of a [new logo](https://matrix.tomesh.net/_matrix/media/v1/download/tomesh.net/yszGAgAETuPkMkzMUZnoMchI)
+- Discussed whether to adopt the new logo, or use the old logo permanently **could not consense on either**
+  - concerns about appearance of new logo, lack of clarity as a 'Toronto' symbol, consistent brand identity
```

Comment:
Ahh I must've missed that, parallel conversations going on my side of the table! Looks good to merge then :)